application/routes/web.php

```diff
@@ -21,7 +21,10 @@
 Route::post('wish_products/{product}', 'WishProductController@store');
 Route::delete('wish_products/{product}', 'WishProductController@destroy');
 Route::resource('users', 'UserController')->only(['edit', 'update']);
-Route::resource('products.product_reviews', 'ProductReviewController');
+
+Route::group(['middleware' => ['auth']], function() {
```

Comment:
定数宣言は補完が効かず記載ミスの元
以下を使用する
```suggestion
Route::middleware('auth')->group(function() {
```